debian/control

```diff
@@ -0,0 +1,12 @@
+Source: ocf-atool
+Section: python
+Priority: extra
+Maintainer: Open Computing Facility <help@ocf.berkeley.edu>
+Build-Depends: debhelper (>= 9), python3, dh-virtualenv, dh-systemd (>= 1.5)
+Standards-Version: 3.9.5
```

Comment:
You could use current version 3.9.6: [changelog](http://metadata.ftp-master.debian.org/changelogs/main/d/debian-policy/debian-policy_3.9.6.1_changelog)